discord/install.sh

```diff
@@ -1,16 +1,17 @@
 #!/bin/bash
 
+sed -i "/#\$nrconf{restart} = 'i';/s/.*/\$nrconf{restart} = 'a';/" /etc/needrestart/needrestart.conf
```

Comment:
Do any of the apps require needrestart?

Reply:
This was to prevent the user interface I showed you from popping up during the install scripts. When doing an apt upgrade, it would ask me which systems I wanted to restart. The default mode for this is interactive, which I don't want, so this line will change it from interactive to automatic.

apt upgrade uses it